src/platform/graphics/webgpu/webgpu-texture.js

```diff
@@ -265,13 +266,19 @@ class WebgpuTexture {
             mipLevel: 0
         };
 
-        // TODO: RGBA only for now, needs to be more generic
-        const numElementsPerPixel = 4;
+        // TODO: handle update to
```

Comment:
Did you mean 'too'?

Reply:
fixed. I was in the middle of typing the comment, jumped elsewhere to make some other change, and never finished it :)